src/main/java/frc/robot/targeting/VisionOutputs.java

```diff
@@ -0,0 +1,47 @@
+package frc.robot.targeting;
+
+import edu.wpi.first.math.geometry.Pose2d;
+
+public class VisionOutputs {
+    private Pose2d tagPosesRelativeToCamera;
```

Comment:
I think we still have the problem that this will only allow us to send one target. What if we have two? 

Consider something like:

List<Pose2d> targetPoses = new ArrayList<>();

this will allow you to send multiple poses. 

I would advise re-implementing hasTargets() to check if this is > 0